java/openmldb-taskmanager/src/main/resources/log4j.properties

```diff
@@ -7,5 +7,5 @@ log4j.appender.file=org.apache.log4j.DailyRollingFileAppender
 log4j.appender.file.append=true
 log4j.appender.file.layout=org.apache.log4j.PatternLayout
 log4j.appender.file.layout.ConversionPattern=%d %p [%c] - %m%n
-log4j.appender.file.file=../log/taskmanager.log
+log4j.appender.file.file=./log/taskmanager.log
```

Comment:
the default log dir of nameserver and tablet is logs. It's better to be consistent with ns/tablet.

Reply:
Okay, I will change to `logs`.